question_3.rb

```diff
@@ -0,0 +1,7 @@
+# ('a'..'z') -> Range of letters 
+# to_a -> creates a array of letters
+# reverse -> as name suggests, it reverses the array
+
+for i in ('a'..'z').to_a.reverse
```

Comment:
instead of having to_a and reverse you can use reverse_each. it is more efficient.